std/string.d

```diff
@@ -62,19 +80,55 @@ strings, use $(D char[]), $(D wchar[]), or $(D dchar[]). The $(D xxxstring)
 types are preferable because they don't exhibit undesired aliasing, thus
 making code more robust.
 
+The following functions are publicly imported:
+
+$(BOOKTABLE ,
+$(TR $(TH Module) $(TH Functions) )
+    $(TR $(TD algorithm)
+        $(TD
+		 $(XREF algorithm, cmp)
+		 $(XREF algorithm, count)
+		 $(XREF algorithm, endsWith)
+		 $(XREF algorithm, startsWith)
+    ))
+    $(TR $(TD array)
+        $(TD
+		 $(XREF array, join)
+		 $(XREF array, split)
+    ))
+    $(TR $(TD format)
+        $(TD
+		 $(XREF format, format)
+		 $(XREF format, sformat)
+    ))
+    $(TR $(TD uni)
+        $(TD
+		 $(XREF uni, icmp)
+		 $(XREF uni, toLower)
+		 $(XREF uni, toLowerInPlace)
+		 $(XREF uni, toUpper)
+		 $(XREF uni, toUpperInPlace)
+    ))
+)
+
 Not all functions for D _string handling are in this module. Functions related
```

Comment:
Oh I see what you mean. I just thought it was a bit awkward that we have already listed publicly imported symbols, which are obviously from elsewhere, and then repeat ourselves that not every symbol is defined in this module. But you do have a good point. Perhaps an even better way is to change this sentence to read something like "There are other string handling functions that are defined in other modules: ...".